treeherder/perf/auto_perf_sheriffing/backfill_tool.py

```diff
@@ -35,7 +48,7 @@ def backfill_job(self, job: Union[Job, str]) -> str:
             },
             root_url=job.repository.tc_root_url,
         )
-        return task_id
+        return (task_id, side_by_side_task_id, )
```

Comment:
If `side_by_side_task_id` is not yet used further, probably best to not add it to the return statement.